packages/api/wrangler.toml

```diff
@@ -43,7 +43,7 @@ bindings = [{ name = "NAME_ROOM", class_name = "NameRoom0" }]
 account_id = "fffa4b4363a7e5250af8357087263b3a" # Protocol Labs CF account
 zone_id = "7eee3323c1b35b6650568604c65f441e"    # web3.storage zone
 route = "https://api.web3.storage/*"
-vars = { CLUSTER_API_URL = "https://web3.storage.ipfscluster.io/api/", ENV = "production", PG_REST_URL = "https://web3-storage-pgrest-prod.herokuapp.com" }
+vars = { CLUSTER_API_URL = "https://web3.storage.ipfscluster.io/api/", ENV = "production", PG_REST_URL = "https://web3-storage-pgrest-prod.herokuapp.com", GATEWAY_URL = "https://ipfs.io" }
```

Comment:
Can we add this to staging and other envs?

Reply:
Good catch!
We definitively need staging to point to [https://ipfs.io](https://ipfs.io)

For other cloudfare dev envs I would suggest we keep the [https://ipfs.io](https://ipfs.io) (rather than the local version) to avoid having to create another tunnel, would you agree?

Honestly, I expect now that we have miniflare, that devs would use that for development rather than this (in most cases) so it shouldn't be a big deal, would you agree?

Reply:
Yes, let's point to ipfs.io

Reply:
Cool, ✅